docs/examples/rx/03_listen_to_balance_change/index.js

```diff
@@ -0,0 +1,38 @@
+// Import the API Rx
+const { ApiRx } = require('@polkadot/api');
+const { WsProvider } = require('@polkadot/rpc-provider');
+
+const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+
+// Known account we want to use (available on dev chain, with funds)
+const Alice = '5GoKvZWG5ZPYL1WUovuHW3zJBWBP5eT8CbqjdRY4Q6iMaDtZ';
+
+async function main () {
+  let previous = null;
+
+  // Here we pass the (optional) provider
+  ApiRx.create(wsProvider).subscribe((api) => {
```

Comment:
Same thing, avoid nested subscribes.

```javascript
ApiRx
  .create(wsProvider)
  .pipe(
    switchMap((api)=> api.query.balances.freeBalance(Alice))
  ) 
```

If you want to add some console.logs, add either in subscribe, or on the `tap` operator.